yb-voyager/src/utils/utils.go

```diff
@@ -130,6 +130,29 @@ func GetSchemaObjectList(sourceDBType string) []string {
 	return requiredList
 }
 
+func ApplyExportSchemaObjectListFilter(allowedObjects *[]string, expectedObjects string) {
+	if expectedObjects == "" {
+		return
+	}
+	expectedObjectsSlice := strings.Split(expectedObjects, ",")
```

Comment:
You should not require this when you have two distinct variables `StrExportObjectTypeList` and `ExportObjectTypeList` at the top level. Then this function can directly take `expectedObjects` of type `[]string`